## 50% of end-end tests have passed
```
Total executed tests: 14
Failed tests: 5
Expected to fail tests: 2
Passed tests: 7
Not executed tests: 2
```
```
IDs of not executed tests: 742015,742016 
```
```
IDs of failed tests: 727230,727231,740490,703133,702843 
```
```
IDs of expected to fail tests: 741840,741841 
```
<h3>Not executed tests (2)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=seanstrom/remove-reagent-override&PR_ID=22320&APK_NAME=22320.apk&TR_CASE_IDS=742015,742016">Rerun not executed tests</a></li></details><h3>Failed tests (5)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=seanstrom/remove-reagent-override&PR_ID=22320&APK_NAME=22320.apk&TR_CASE_IDS=727230,727231,740490,703133,702843">Rerun failed tests</a></li><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3823803">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div># STEP: Check that removed user is not shown in the list anymore</div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:246: in test_restore_multiaccount_with_waku_backup_remove_profile_switch
    self.errors.verify_no_errors()
base_test_case.py:179: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: zQ3...dWXh5 was not restored as a contact from waku backup!
E    Device 1: zQ3...Vacac was not restored as a contact from waku backup!
E    Device 1: admin_open was not restored from waku-backup!!
E    Device 1: member_open was not restored from waku-backup!!
E    Device 1: admin_closed was not restored from waku-backup!!
E    Device 1: member_closed was not restored from waku-backup!!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuOxRVStsQMozFCkG">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3823806">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><p><blockquote><div># STEP: Adding new regular account</div><div>Device 1: Find `Button` by `accessibility id`: `add-account` </div></blockquote></p>

```
critical/wallet/test_wallet_mainnet.py:357: in test_wallet_add_remove_regular_account
    self.wallet_view.add_regular_account(account_name=new_account_name)
../views/wallet_view.py:245: in add_regular_account
    self.add_account_button.click()
../views/base_element.py:89: in click
    element = self.find_element()
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `add-account` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3823807">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `network-dropdown` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/wallet/test_wallet_mainnet.py:70: in test_wallet_balance_mainnet
    self.errors.verify_no_errors()
base_test_case.py:179: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: For the USD Coin the wrong value 0.0 is shown, expected 5.0 in total
E    Device 1: For the USD Coin the wrong value 0.0 is shown, expected 5.0 on Optimism
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3823802">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'Message AFTER edit 2 (Edited)')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-2']/android.widget.TextView[2]` </div><div>Device 1: Element EmojisNumber text is equal to 1 </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:388: in test_community_message_edit
    self.errors.verify_no_errors()
base_test_case.py:179: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 2: Message is not edited
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESESRjSJwDR33HQiVl">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES3x4WvI7x4lFeGweh">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3823805">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='container']/android.widget.TextView[@text='Ether']/../android.widget.TextView[3]` </div><div>Device 1: `Text` is `0.24231 ETH` </div></blockquote></p>

```
critical/wallet/test_wallet_testnet.py:192: in test_wallet_send_asset_from_drawer
    self._check_balances_after_tx(amount_to_send, None, None, eth_amount_sender, eth_amount_receiver)
critical/wallet/test_wallet_testnet.py:96: in _check_balances_after_tx
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
critical/wallet/test_wallet_testnet.py:73: in wait_for_wallet_balance_to_update
    new_eth_amount = round(wallet_view.get_asset(asset_name='Ether').get_amount(), 4)
../views/wallet_view.py:193: in get_asset
    element.scroll_to_element(down_start_y=0.89, down_end_y=0.8)
../views/base_element.py:188: in scroll_to_element
    return self.find_element()
../views/base_element.py:83: in find_element
    raise exception
../views/base_element.py:76: in find_element
    return self.driver.find_element(self.by, self.locator)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/root/lambda/shared-tools/volume/uiautomator/uiautomator-2.32.3/node_modules/@appium/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at runMicrotasks (<anonymous>)
E       at processTicksAndRejections (node:internal/process/task_queues:96:5)
E       at AndroidUiautomator2Driver.doFindElementOrEls (/root/lambda/shared-tools/volume/uiautomator/uiautomator-2.32.3/lib/commands/find.js:44:7)
E       at doFind (/root/lambda/shared-tools/volume/uiautomator/uiautomator-2.32.3/node_modules/appium-android-driver/lib/commands/find.ts:48:17)
E       at wrappedCondFn (/root/lambda/shared-tools/volume/uiautomator/uiautomator-2.32.3/node_modules/@appium/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/root/lambda/shared-tools/volume/uiautomator/uiautomator-2.32.3/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/root/lambda/shared-tools/volume/uiautomator/uiautomator-2.32.3/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/root/lambda/shared-tools/volume/uiautomator/uiautomator-2.32.3/node_modules/@appium/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/root/lambda/shared-tools/volume/uiautomator/uiautomator-2.32.3/node_modules/appium-android-driver/lib/commands/find.ts:71:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/root/lambda/shared-tools/volume/uiautomator/uiautomator-2.32.3/node_modules/@appium/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/root/lambda/shared-tools/volume/uiautomator/uiautomator-2.32.3/node_modules/@appium/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/></td></tr></tbody></table></details><h3>Expected to fail tests (2)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletCollectibles:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3823812">test_wallet_send_collectible</a>, id: 741840 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `EditBox` by `accessibility id`: `address-text-input` </div></blockquote></p>

```
critical/wallet/test_collectibles.py:102: in test_wallet_send_collectible
    self.wallet_view.address_text_input.send_keys(self.receiver['wallet_address'])
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `address-text-input` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception 
```

<b>[[Collectibles list is not loaded sometimes]]</b><br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3823813">test_wallet_collectible_send_from_expanded_info_view</a>, id: 741841 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `collectibles-tab` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/wallet/test_collectibles.py:158: in test_wallet_collectible_send_from_expanded_info_view
    self.wallet_view.get_collectible_element('Glitch Punks').wait_for_element().click()
../views/base_element.py:120: in wait_for_element
    raise TimeoutException(
 Device `1`: `CollectibleItemElement` by` xpath`: `//*[@content-desc='collectible-list-item']//*[contains(@text,'Glitch Punks')]/../..` is not found on the screen after wait_for_element 
```

<b>[[Collectibles list is not loaded sometimes]]</b><br/><br/></td></tr></tbody></table></details><h3>Passed tests (7)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3823809">test_wallet_swap_flow_mainnet</a>, id: 741555 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3823810">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3823808">test_wallet_send_flow_mainnet</a>, id: 741554 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestWalletCollectibles:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3823811">test_wallet_collectibles_balance</a>, id: 741839 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3823800">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuOxRVStsQMozFCkG">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3823801">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESTIiklA4vFL6Ax1yT">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESS9UPo1R8uVAHlrPr">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3823804">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table></details>

--- a/src/status_im/core.cljs
+++ b/src/status_im/core.cljs
@@ -35,6 +35,20 @@
 ;;;; re-frame RN setup
 (set! interop/next-tick js/setTimeout)
 
+;; Note: In the past we've configured reagent to run its batch rendering
+;; faster by overriding the next-tick function. This technique could be useful
+;; if we want to adjust the batch speed for different frame-rates since by
+;; default reagent tunes its batch rendering for 60FPS.
+;;
+;; For example, this code would have batches rendering as fast as possible
+;; (set! reagent.impl.batching/next-tick js/setImmediate)
+;;
+;; While this example would approximate 120FPS
+;; (set! reagent.impl.batching/next-tick #(js/setTimeout % 8))
+;;
+;; And under the hood this is what reagent will use for approximately 60FPS:
+;; (set! reagent.impl.batching/next-tick #(js/setTimeout % 16))
+
 (def adjust-resize 16)
 
 (defn is-hermes

--- a/src/status_im/core.cljs
+++ b/src/status_im/core.cljs
@@ -16,7 +16,6 @@
     [react-native.platform :as platform]
     [react-native.shake :as react-native-shake]
     [reagent.core]
-    [reagent.impl.batching :as batching]
     [status-im.common.log :as logging]
     [taoensso.timbre :as log]
     [status-im.common.universal-links :as universal-links]
@@ -35,7 +34,6 @@
 
 ;;;; re-frame RN setup
 (set! interop/next-tick js/setTimeout)
-(set! batching/fake-raf #(js/setTimeout % 0))
 
 (def adjust-resize 16)
 

